ntbs-service/Migrations/20191204102129_AddSocialContextVenues.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;

```

Comment:
Sorry for not splitting this migration into several - my snapshot was out of sync somehow and I had to manually delete old migrations and add one again which then contained all the changes...